administrator/components/com_associations/models/fields/menuparent.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_menus
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('JPATH_BASE') or die;
+
+JFormHelper::loadFieldClass('list');
```

Comment:
oh I read that wrong, since you commented this line:
`JFormHelper::loadFieldClass('list');`I thought you were referring to it, this file should not be here indeed, I'll remove it